content/post/frontend_talks_2017.md

```diff
@@ -0,0 +1,142 @@
+---
+author: erizocosmico
+date: 2017-07-10
+title: "source{d} tech talks, frontend series"
+image: /post/frontend-talks/header-pic.jpg
+description: "Once every few months, source{d} organizes small conferences around a very specific topic. On June, 24th the topic was frontend and the talks were hosted in our Madrid offices. Almost 50 people joined us for a day full of things to learn about frontend technologies."
+categories: ["technical", "frontend"]
+---
+
+![Schedule with our logo](/post/frontend-talks/header-full.jpg)
+
+Once every few months, source{d} organizes small conferences around a very specific topic. On June, 24th the topic was frontend and the talks were hosted in our Madrid offices. Almost 50 people joined us for a day full of things to learn about frontend technologies.
+
+The topics of the talks were very diverse, from animations with Ember to multi-threaded web to React styiling to accessibility. Since frontend is such a wide area in programming, we aimed to provide talks from a lot of different topics, which we thing we accomplished.
+
+## Conferences are no fun on an empty stomach
+
+First things first, we started the day with a nice welcome breakfast for all the attendees and speakers in which people could talk and get to know each other as well as the speakers before their talks.
+
+## Opening speech
+
+After everyone was in their seat, [Margarida García](https://twitter.com/margaridagsl) gave the opening talk and welcomed all the people in the room. She talked a bit about what we do here at source{d} and explained what the source{d} tech talks was about and why the community was so important to us.
+
+## Talks
+
+The day was split in two sessions: the morning and the afternoon sessions. The morning session had 3 main talks and the afternoon session only two, having the morning session a lightning talk at the end of it and another lightning talk at the start of the afternoon session. Due to a last-minute cancellation of a speaker the lightning talk from the afternoon session was removed.
+Between sessions there was a lunch break and plenty of coffee breaks during each of the sessions.
```

Comment:
Between each talk we had a couple of breaks to allow everyone to restore energies and had a larger one at lunch.